src/com/projectkorra/projectkorra/configuration/ConfigManager.java

```diff
@@ -447,87 +447,102 @@ public static void configCheck(final ConfigType type) {
 			config = defaultConfig.get();
 
 			final ArrayList<String> earthBlocks = new ArrayList<String>();
+			earthBlocks.add("#base_stone_overworld");
+			earthBlocks.add("#base_stone_nether");
+			earthBlocks.add("#coal_ores"); // added in 1.17
+			earthBlocks.add("#lapis_ores"); // added in 1.17
+			earthBlocks.add("#redstone_ores"); // added in 1.17
+			earthBlocks.add("#emerald_ores"); // added in 1.17
+			earthBlocks.add("#diamond_ores"); // added in 1.17
 			earthBlocks.add(Material.DIRT.toString());
 			earthBlocks.add(Material.COARSE_DIRT.toString());
 			earthBlocks.add(Material.PODZOL.toString());
 			earthBlocks.add(Material.MYCELIUM.toString());
-			earthBlocks.add(Material.STONE.toString());
 			earthBlocks.add(Material.GRAVEL.toString());
 			earthBlocks.add(Material.CLAY.toString());
-			earthBlocks.add(Material.COAL_ORE.toString());
-			earthBlocks.add(Material.REDSTONE_ORE.toString());
-			earthBlocks.add(Material.LAPIS_ORE.toString());
-			earthBlocks.add(Material.DIAMOND_ORE.toString());
-			earthBlocks.add(Material.EMERALD_ORE.toString());
-			earthBlocks.add(Material.NETHERRACK.toString());
 			earthBlocks.add(Material.COBBLESTONE.toString());
 			earthBlocks.add(Material.STONE_SLAB.toString());
 			earthBlocks.add(Material.COBBLESTONE_SLAB.toString());
 			earthBlocks.add(Material.GRASS_BLOCK.toString());
 			earthBlocks.add(Material.GRASS_PATH.toString());
-			earthBlocks.add(Material.ANDESITE.toString());
-			earthBlocks.add(Material.GRANITE.toString());
-			earthBlocks.add(Material.DIORITE.toString());
-			earthBlocks.add(Material.BASALT.toString());
 			earthBlocks.add(Material.ANCIENT_DEBRIS.toString());
-			earthBlocks.add(Material.BLACKSTONE.toString());
+			earthBlocks.add(Material.COAL_ORE.toString()); // remove in 1.17
+			earthBlocks.add(Material.LAPIS_ORE.toString()); // remove in 1.17
+			earthBlocks.add(Material.REDSTONE_ORE.toString()); // remove in 1.17
+			earthBlocks.add(Material.DIAMOND_ORE.toString()); // remove in 1.17
+			earthBlocks.add(Material.EMERALD_ORE.toString()); // remove in 1.17
+			earthBlocks.add("SMALL_AMETHYST_BUD");
+			earthBlocks.add("MEDIUM_AMETHYST_BUD");
+			earthBlocks.add("LARGE_AMETHYST_BUD");
+			earthBlocks.add("DRIPSTONE_BLOCK");
+			earthBlocks.add("CALCITE");
 
 			final ArrayList<String> metalBlocks = new ArrayList<String>();
-			metalBlocks.add(Material.IRON_ORE.toString());
-			metalBlocks.add(Material.GOLD_ORE.toString());
+			metalBlocks.add("#copper_ores"); // added in 1.17
+			metalBlocks.add("#iron_ores"); // added in 1.17
+			metalBlocks.add("#piglin_loved"); // includes all gold ores/blocks

```

Comment:
Yeah of course, feel free to use whatever tags YOU guys feel would be best suited in the default config and feel free to trim the 1.17 materials as well. I was just here to make it possible because I didn't see any reason why it shouldn't be. #gold_ores happens to be the only ore group present in 1.16 so feel free to use that instead.

Reply:
Looks like the following would be the blocks we would want:
- #gold_ores
- gold_block
- gilded_blackstone
- raw_gold_block

light_weighted_pressure_plate and bell are up for debate 😆

Reply:
I was wondering about the plates tbh. lol I'll clean this up tonight with your suggestions. Thank you!